test/e2e/network_policy_helpers.go

```
@@ -0,0 +1,829 @@
+					Command: []string{"/agnhost"},
+					Args:    []string{"netexec", fmt.Sprintf("--http-port=%d", port)},
+					Ports: []corev1.ContainerPort{
+						{ContainerPort: port},
+					},
+				},
+			},
+		},
+	}
+}
+
+
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofumpt)